presto-native-execution/scripts/setup-centos.sh

```diff
@@ -17,42 +17,68 @@ set -x
 export FB_OS_VERSION=v2024.04.01.00
 export RE2_VERSION=2021-04-01
 export nproc=$(getconf _NPROCESSORS_ONLN)
-
-dnf install -y maven java python3-devel clang-tools-extra jq perl-XML-XPath
-
-python3 -m pip install regex pyyaml chevron black
-
 export CC=/opt/rh/gcc-toolset-9/root/bin/gcc
 export CXX=/opt/rh/gcc-toolset-9/root/bin/g++
 
 CPU_TARGET="${CPU_TARGET:-avx}"
 SCRIPT_DIR=$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")
-if [ -f "${SCRIPT_DIR}/setup-helper-functions.sh" ]
+if [ -f "${SCRIPT_DIR}/setup-centos8.sh" ]
 then
-  source "${SCRIPT_DIR}/setup-helper-functions.sh"
+  source "${SCRIPT_DIR}/setup-centos8.sh"
 else
-  source "${SCRIPT_DIR}/../velox/scripts/setup-helper-functions.sh"
+  source "${SCRIPT_DIR}/../velox/scripts/setup-centos8.sh"
 fi
 
-export COMPILER_FLAGS=$(echo -n $(get_cxx_flags $CPU_TARGET))
+function install_presto_deps_from_package_managers {
+  dnf install -y maven java clang-tools-extra jq perl-XML-XPath
+  # This python version is installed by the Velox setup scripts
+  pip3.9 install regex pyyaml chevron black
+}
 
-(
+function install_gperf {
   wget http://ftp.gnu.org/pub/gnu/gperf/gperf-3.1.tar.gz &&
   tar xvfz gperf-3.1.tar.gz &&
   cd gperf-3.1 &&
   ./configure --prefix=/usr/local/gperf/3_1 &&
   make "-j$(nproc)" &&
   make install &&
   ln -s /usr/local/gperf/3_1/bin/gperf /usr/local/bin/
```

Comment:
Can we include a line about removing the symlink if it exists?
I've run into the problem where the symlink exists and line 45 will fail aborting the processing.

Reply:
Good point. I will add a check if the target exists.